src/kibana/plugins/settings/sections/objects/_objects.html

```diff
@@ -1,6 +1,10 @@
 <kbn-settings-app section="objects">
   <kbn-settings-objects class="container">
-    <h2>Edit Saved Objects</h2>
+    <div class="header">
+      <h2 class="title">Edit Saved Objects</h2>
+      <a class="controls" ng-click="exportAll()"><i aria-hidden="true" class="fa fa-download"></i> Export</a>
```

Comment:
IMO, `<a>` tags should use href and not click handlers; `<buttons>` are more semantic in this scenario. A button style that resembles a link may be worth creating.